src/editorial-guidelines.tm

```diff
@@ -71,15 +76,15 @@
 
 <\references>
   <\collection>
-    <associate|auto-1|<tuple|?|?|contribution-guide.tm>>
+    <associate|auto-1|<tuple|?|?>>
   </collection>
 </references>
 
 <\auxiliary>
   <\collection>
     <\associate|toc>
-      <vspace*|1fn><with|font-series|<quote|bold>|math-font-series|<quote|bold>|font-shape|<quote|small-caps>|Contribution
-      guide> <datoms|<macro|x|<repeat|<arg|x>|<with|font-series|medium|<with|font-size|1|<space|0.2fn>.<space|0.2fn>>>>>|<htab|5mm>>
+      <vspace*|1fn><with|font-series|<quote|bold>|math-font-series|<quote|bold>|font-shape|<quote|small-caps>|Editorial
+      scope and author guidelines> <datoms|<macro|x|<repeat|<arg|x>|<with|font-series|medium|<with|font-size|1|<space|0.2fn>.<space|0.2fn>>>>>|<htab|5mm>>
```

Comment:
This deletion/addition is probably due to me missing the new sentence when merging files, as I do not remember having changed this sentence.